Assets/Scripts/Editor/ParallaxEffect/PopupWindow/ConfirmDelete.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UIElements;
+using UnityEditor;
+using System;
+
+public class ConfirmDelete : PopupWindowContent
+{
+    public event Action<ConfirmDelete> OnConfirm;
+    public event Action<ConfirmDelete> OnCancel;
+    public event Action<ConfirmDelete> OnCloseWindow;
+
+    public override Vector2 GetWindowSize()
+    {
+        return new Vector2(200, 100);
+    }
+
+    public override void OnGUI(Rect rect)
+    {
+    }
```

Comment:
empty OnGUI function

Reply:
This method is only here, because class inherits from PopupWindowContent and it needs to implement the OnGUI method

Reply:
ahh makes sense